2996_reflection/reflection.md

```diff
@@ -8060,6 +8060,16 @@ Our framework for code injection as performed by `define_aggregate` evolved quit
 
 ---
 references:
+  - id: P2830R10
+    citation-label: P2830R10
+    title: "Constexpr Type Ordering"
+    author:
+      - family: Nate Nichols, Gašper Ažman
+    issued:
+      year: 2025
+      month: 2
+      day: 21
+    URL: https://wg21.link/p2830r10
```

Comment:
This revision is already published, shouldn't be necessary. Did you do `make update`?

Reply:
You know the answer.